content/applications/marketing/surveys/dashboard.rst

```diff
@@ -0,0 +1,178 @@
+measure the satisfaction of customers and/or employees. This process yields valuable insights into
+shifting market trends.
+
+Dashboard
+=========
+
+Upon opening the *Surveys* application, Odoo presents the main dashboard of the *Surveys*
+application, otherwise known as the :guilabel:`Surveys` page.
+
+.. image:: dashboard/surveys-dashboard.png
+   :align: center
+   :alt: The Odoo Surveys application dashboard in the default kanban view.
+
+.. tip::
+   The *Surveys* dashboard can be accessed at any time throughout the application by clicking
+   :menuselection:`Surveys` from the header menu.
+
+In the upper-left corner, there is a :guilabel:`New` button. When clicked, Odoo presents a blank
+survey form that can be used to :doc:`create a survey <../surveys/create>`.
+
+On the dashboard, all the surveys that have been created in the database are displayed in a default
+Kanban view.
+
+From left-to-right, after the survey name, the user who is responsible for it, and the month it was
+created, each line on the *Surveys* dashboard shows the following:
+
+- Number of :guilabel:`Questions` in that particular survey
+- :guilabel:`Average Duration` of the survey (how long it typically takes a participant to complete)
+- Number of :guilabel:`Registered` participants for the survey
+- Number of times that particular survey has been :guilabel:`Completed`
+- Percentage and data bar showcasing how many people have :guilabel:`Passed` or become
+  :guilabel:`Certified`
+
+.. note::
+   A half-trophy background image behind the survey name indicates that the survey is a
+   *Certification*.
+
+.. note::
+   The :guilabel:`Passed` percentage and bar **only** appears when a *Required Score* has been
+   configured for that particular survey.
+
+   The :guilabel:`Certfied` percentage and bar **only** appears if that particular survey has the
+   *Is a Certification* option enabled on the survey form.
+
+   If neither :guilabel:`Passed` nor :guilabel:`Certified` appear on the line, that indicates the
+   survey is without a *Required Score* and was not enabled with the *Is a Certification* option.
+
+- Number of :guilabel:`Courses` related to that survey, which **only** appears if more than one
+  course has been created and attached to a single survey
+
+To the far-right of those data points on the survey lines, located on the *Surveys* application
+dashboard, are a collection of buttons.
+
+Those buttons are as follows:
+
+- :guilabel:`Share`: click to reveal a :guilabel:`Share a Survey` pop-up form that can be used to
+  invite potential participants to the survey - complete with a :guilabel:`Survey Link` that can be
+  copied and sent to potential participants, and a :guilabel:`Send by Email` toggle switch.
+
+   .. image:: dashboard/share-survey-popup.png
+      :align: center
+      :alt: The 'Share a Survey' pop-up window that appears in the Odoo Surveys application.
```

Comment:
This image is not needed since the following displays what this image shows, as well as the advanced configuration.

Also update terminology, this "pop-up" is called a "modal" :)
```suggestion
- :guilabel:`Share`: click to reveal a :guilabel:`Share a Survey` modal that can be used to invite
  potential participants to the survey - complete with a :guilabel:`Survey Link` that can be copied
  and sent to potential participants, and a :guilabel:`Send by Email` toggle switch.
```

Reply:
will delete the image, but am disregarding the "modal" suggestion, per our recent decision re: "modal" vs. "pop-up..."